src/main/resources/config.properties

```diff
@@ -1,5 +1,5 @@
8438.32859"},"windowsCompatible":true,"linuxCompatible":false,"description":"This is the windows CLI.","feedUrl":"https:\/\/uipath.pkgs.visualstudio.com\/Public.Feeds\/_packaging\/UiPath-Official\/nuget\/v3\/flat2\/UiPath.CLI.Windows\/22.10.8438.32859\/UiPath.CLI.Windows.22.10.8438.32859.nupkg"},{"displayName":"UiPath CrossPlatform CLI (ver.23.2.8467.25277)","name":"UiPath.CLI","version":{"major":"23","minor":"2","patch":"25277","complete":"23.2.8467.25277"},"windowsCompatible":false,"linuxCompatible":true,"description":"This is the crossplatform CLI.","feedUrl":"https:\/\/uipath.pkgs.visualstudio.com\/Public.Feeds\/_packaging\/UiPath-Official\/nuget\/v3\/flat2\/UiPath.CLI\/23.2.8467.25277\/UiPath.CLI.23.2.8467.25277.nupkg"},{"displayName":"UiPath Windows CLI (ver.23.2.8467.25277)","name":"UiPath.CLI.Windows","version":{"major":"23","minor":"2","patch":"25277","complete":"23.2.8467.25277"},"windowsCompatible":true,"linuxCompatible":false,"description":"This is the windows CLI.","feedUrl":"https:\/\/uipath.pkgs.visualstudio.com\/Public.Feeds\/_packaging\/UiPath-Official\/nuget\/v3\/flat2\/UiPath.CLI.Windows\/23.2.8467.25277\/UiPath.CLI.Windows.23.2.8467.25277.nupkg"},{"displayName":"UiPath CrossPlatform CLI (ver.23.6.8581.19168)","name":"UiPath.CLI","version":{"major":"23","minor":"6","patch":"19168","complete":"23.6.8581.19168"},"windowsCompatible":false,"linuxCompatible":true,"description":"This is the crossplatform CLI.","feedUrl":"https:\/\/uipath.pkgs.visualstudio.com\/Public.Feeds\/_packaging\/UiPath-Official\/nuget\/v3\/flat2\/UiPath.CLI\/23.6.8581.19168\/UiPath.CLI.23.6.8581.19168.nupkg"},{"displayName":"UiPath Windows CLI (ver.23.6.8581.19168)","name":"UiPath.CLI.Windows","version":{"major":"23","minor":"6","patch":"19168","complete":"23.6.8581.19168"},"windowsCompatible":true,"linuxCompatible":false,"description":"This is the windows CLI.","feedUrl":"https:\/\/uipath.pkgs.visualstudio.com\/Public.Feeds\/_packaging\/UiPath-Official\/nuget\/v3\/flat2\/UiPath.CLI.Windows\/23.6.8581.19168\/UiPath.CLI.Windows.23.6.8581.19168.nupkg"},{"displayName":"UiPath CrossPlatform CLI (ver.23.10.8748.26221)","name":"UiPath.CLI","version":{"major":"23","minor":"10","patch":"26221","complete":"23.10.8748.26221"},"windowsCompatible":false,"linuxCompatible":true,"description":"This is the crossplatform CLI.","feedUrl":"https:\/\/uipath.pkgs.visualstudio.com\/Public.Feeds\/_packaging\/UiPath-Official\/nuget\/v3\/flat2\/UiPath.CLI\/23.10.8748.26221\/UiPath.CLI.23.10.8748.26221.nupkg"},{"displayName":"UiPath Windows CLI (ver.23.10.8748.26221)","name":"UiPath.CLI.Windows","version":{"major":"23","minor":"10","patch":"26221","complete":"23.10.8748.26221"},"windowsCompatible":true,"linuxCompatible":false,"description":"This is the default pre-packaged windows CLI version.","feedUrl":"https:\/\/uipath.pkgs.visualstudio.com\/Public.Feeds\/_packaging\/UiPath-Official\/nuget\/v3\/flat2\/UiPath.CLI.Windows\/23.10.8748.26221\/UiPath.CLI.Windows.23.10.8748.26221.nupkg"}]
```

Comment:
It's difficult to understand what has been modified on this property. This looks like a json. Can we put it on multiple lines, so it's easy to read?

Reply:
We can try how it works as an improvement. For now, it will postpone release pipeline executions.